app/index.html

```diff
@@ -20,7 +18,7 @@
 		}
 	</script>
 
-	<main>Loading...</main>
+	<main></main>
```

Comment:
Recommend retaining this `Loading...`.  In case the script loads slowly, this lets the user know that something is happening while they wait for Preact's first render.

Reply:
Was going to build a fancier one but yeah, will move it to another PR anyways so reverted for now